src/seedu/addressbook/commands/HelpCommand.java

```diff
@@ -21,6 +21,7 @@ public CommandResult execute() {
                 + "\n" + ListCommand.MESSAGE_USAGE
                 + "\n" + ViewCommand.MESSAGE_USAGE
                 + "\n" + ViewAllCommand.MESSAGE_USAGE
+                + "\n" + AboutCommand.MESSAGE_USAGE
```

Comment:
Should the about command be the first command that is listed in the list?

Reply:
Mmm, no strong opinion on this one, AboutCommand is placed here deliberately above HelpCommand since they both serve similar purposes and are probably less looked for. Although this does break the alphabetical ordering.